Automattic/wp-calypso
# HappyBlocks: Adapt header for learn website

Related to https://github.com/Automattic/wpsupport3/pull/478

## Proposed Changes

Adapts `happyblocks` header for [learn](https://learncft.wordpress.com/) website
[Design reference](IceKRdz5qlPYwuqzu6pBgF-fi-3733%3A95781)

## Testing Instructions

- Sandbox `learn.wordpress.com` and `wordpress.com`
- Pull this branch 
- Navigate to `/wp-calypso/apps/happy-blocks` and run `yarn dev --sync` to sync with your sandbox
- Open `learncft.wordpress.com`
- It should match the design.
- In addition, please check `wordpress.com/support` and make sure nothing has changed there once this change should only be applied to `/learn` website for now.


Learn should look like:

<img width="819" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/a8711672-e318-4abd-b246-95b7b943e4ac">


Support should look like:

<img width="1535" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/71f59963-3de0-449e-9bba-0231fb872b2a">
